src/condor_startd.V6/ResAttributes.cpp

```diff
@@ -521,6 +527,20 @@ MachAttributes::compute_for_policy()
 	}
 }
 
+void 
+MachAttributes::compute_docker_cache() {
+#ifdef LINUX
+	static time_t lastRun = 0;
+	time_t now = time(nullptr);
+	if ((now - lastRun) > 1200) {
```

Comment:
Perhaps this '1200' should be a config knob, and/or at the very least the documentation should state that the docker cache size attribute is updated every 20 minutes

Reply:
Knobbed and documented.